match/lib/match/runner.rb

```diff
@@ -356,7 +356,7 @@ def fetch_provisioning_profile(params: nil, profile_type:, certificate_id: nil,
 
       if params[:output_path]
         FileUtils.cp(stored_profile_path, params[:output_path])
-        installed_profile = FastlaneCore::ProvisioningProfile.install(profile, keychain_path)
+        installed_profile = FastlaneCore::ProvisioningProfile.install(stored_profile_path, keychain_path)
```

Comment:
@nekrich https://github.com/fastlane/fastlane/pull/21691/files#diff-8cd09d7ab3f4398e3160f603839ac2891e3cdf60e7f09930a1288ce33ab8fc1fR359

Looking deeper: I'm a bit less sure about the intent here. It looks to me like we already do a `FastlaneCore::ProvisioningProfile.install` on line 354 just above this: https://github.com/fastlane/fastlane/blob/master/match/lib/match/runner.rb#L354

Is it necessary to run that again if `output_path` is supplied? Wouldn't that be the same profile that was installed on line 354?

If so, is `stored_profile_path` or `params[:output_path]` the correct param?

Reply:
Thanks @TheMetalCode  for investigation 💪 This looks like a typo, but as you noted it doesn't seem to be necessary and could be removed.

Installing profile (copying it to a directory used by Xcode) makes sense only on a Mac OS and it's done in line 354.
Also certificates wouldn't be installed on another operating system 
https://github.com/fastlane/fastlane/blob/152b38262f3a5563c07133555ecc080c4f8e43e0/match/lib/match/runner.rb#L220-L239

Even if we wanted to install the same profile twice, it would be copied to under the same name, as those profiles are named with UUID.

https://github.com/fastlane/fastlane/blob/152b38262f3a5563c07133555ecc080c4f8e43e0/fastlane_core/lib/fastlane_core/provisioning_profile.rb#L75-L88

https://github.com/fastlane/fastlane/blob/152b38262f3a5563c07133555ecc080c4f8e43e0/fastlane_core/lib/fastlane_core/provisioning_profile.rb#L38-L40

https://github.com/fastlane/fastlane/blob/152b38262f3a5563c07133555ecc080c4f8e43e0/fastlane_core/lib/fastlane_core/provisioning_profile.rb#L62-L65

Reply:
@lucgrabowski Appreciate the response! It does seem safe to simply remove the line that was causing the issue, which I've done and have tested successfully